src/applications/vaos/actions/newAppointment.js

```diff
@@ -705,6 +705,7 @@ export function submitAppointmentOrRequest(router) {
         captureError(error, true);
         dispatch({
           type: FORM_SUBMIT_FAILED,
+          isVaos400Error: error?.errors?.[0]?.code === 'VAOS_400',
```

Comment:
Not sure if this 400 can occur for both direct schedule and requests, so I've added it in both error handlers.